tox.ini

```diff
@@ -9,17 +9,24 @@ envlist = py27,
           pypy3,
           coverage,
           docs
+# Tox 2.9.0 is the last version that supports Python 3.3
 minversion = 2.9.0
 skip_missing_interpreters = true
 
 [testenv]
 description = run the unit tests with pytest under {basepython}
 setenv = COVERAGE_FILE={toxworkdir}/.coverage.{envname}
 passenv = DATEUTIL_MAY_CHANGE_TZ TOXENV CI TRAVIS TRAVIS_* APPVEYOR APPVEYOR_* CODECOV_*
-commands = python -m pytest -m "not xfail" {posargs: "{toxinidir}/dateutil/test" --cov-config="{toxinidir}/tox.ini" --cov=dateutil}
+commands = python -m pytest -m "not xfail" --doctest-modules {posargs: "{toxinidir}/dateutil" --cov-config="{toxinidir}/tox.ini" --cov=dateutil}
```

Comment:
IMO `--doctest-glob=*.rst` is also needed, s.t. the doctests in the rst files are run. Note that pytest only runs the tests that don't use the sphinx-directives. We could use pytest-sphinx to run all off them.